backends/platform/sdl/sailfish/rpm/org.scummvm.scummvm.spec

```diff
@@ -0,0 +1,276 @@
+    make install
+    cd ../..
+fi
+if ! [ -d %{outdir}/sdl-install ]; then
+    cd %{outdir}
+    tar xvf ../rpmdeps/SDL2-%{sdl2_version}.tar.gz
+    cd SDL2-%{sdl2_version}
+    for x in ../../sdl-patches/*.patch; do
+	echo "Applying patch $x"
+	patch -p1 -i "$x"
+    done
+    cd ..
+    cmake \
+    -Bsdl-build \
+    -DSDL_PULSEAUDIO=ON \
+    -DSDL_RPATH=OFF \
+    -DSDL_STATIC=ON \
+    -DSDL_SHARED=OFF \
+    -DSDL_WAYLAND=ON \
+    -DSDL_X11=OFF \
+    -DSDL_DBUS=ON \
+    -DSDL_WAYLAND_LIBDECOR=OFF \
+    -DSDL_WAYLAND_QT_TOUCH=OFF \
+    SDL2-%{sdl2_version}
+    make -C sdl-build %{?_smp_mflags}
+    make -C sdl-build install DESTDIR=$PWD/sdl-install INSTALL_ROOT=$PWD/sdl-install
+    cd ..
+fi
+sed "s@Libs: .*@Libs: $PWD/%{outdir}/freetype-install/lib/libfreetype.a@g" < %{outdir}/freetype-install/lib/pkgconfig/freetype2.pc > %{outdir}/pkgconfig/freetype2.pc
+sed "s@Libs: .*@Libs: $PWD/%{outdir}/fribidi-install/lib/libfribidi.a@g" < %{outdir}/fribidi-install/lib/pkgconfig/fribidi.pc > %{outdir}/pkgconfig/fribidi.pc
+export PKG_CONFIG_PATH=$PWD/%{outdir}/pkgconfig:$PKG_CONFIG_PATH
+cd %{outdir}/scummvm;
+../../../../../../configure --host=sailfish \
+		--with-jpeg-prefix=../jpeg-install/usr \
+		--with-mad-prefix=../libmad-install/usr/local \
+		--with-flac-prefix=../flac-install/usr/local \
+		--with-theoradec-prefix=../theora-install/usr/local \
+		--with-mpeg2-prefix=../mpeg2-install/usr/local \
+		--with-faad-prefix=../faad-install/usr/local \
+		--with-gif-prefix=../giflib-install/usr/local \
+		--enable-fribidi --with-fribidi-prefix=../fribidi-install/usr/local/org.scummvm.scummvm  \
+		--with-sdl-prefix=../sdl-install/usr/local --enable-static \
+		%{engine_config} --disable-tinygl
+cd ../..
+%{__make} -C %{outdir}/scummvm %{_make_output_sync} %{?_smp_mflags}
+
+%install
+rm -rf %{buildroot}/*
+%{__make} -C %{outdir}/scummvm DESTDIR=%{buildroot} INSTALL_ROOT=%{buildroot} install
+# TODO: Move this stuff into make
+mkdir -p %{buildroot}/usr/share/applications
+mkdir -p %{buildroot}/usr/share/icons/hicolor/86x86/apps
+mkdir -p %{buildroot}/usr/share/icons/hicolor/108x108/apps
+mkdir -p %{buildroot}/usr/share/icons/hicolor/128x128/apps
+mkdir -p %{buildroot}/usr/share/icons/hicolor/172x172/apps
+cp ../../../../dists/sailfish/org.scummvm.scummvm.desktop %{buildroot}/usr/share/applications/org.scummvm.scummvm.desktop
+cp ../../../../dists/sailfish/86x86.png %{buildroot}/usr/share/icons/hicolor/86x86/apps/org.scummvm.scummvm.png
+cp ../../../../dists/sailfish/108x108.png %{buildroot}/usr/share/icons/hicolor/108x108/apps/org.scummvm.scummvm.png
+cp ../../../../dists/sailfish/128x128.png %{buildroot}/usr/share/icons/hicolor/128x128/apps/org.scummvm.scummvm.png
+cp ../../../../dists/sailfish/172x172.png %{buildroot}/usr/share/icons/hicolor/172x172/apps/org.scummvm.scummvm.png
+
+%files
+%defattr(755,root,root,-)
+%{_bindir}/org.scummvm.scummvm
+
+%defattr(644,root,root,-)
+
+%{_datadir}/org.scummvm.scummvm/applications/%{name}.desktop
```

Comment:
Is there any way to avoid listing every file manually?

Reply:
Changed to wildcard